mof_array/pmf/process_mass_data.py

```diff
@@ -62,13 +62,14 @@ def write_output_data(filename, data):
     return(writer)
 
 # ----- Convert simulated/experimental data into dictionary format -----
-#    ----- Calculates masses in terms of mg/(cm^3 of framework) -----
+# Also calculates masses in terms of mg/(cm^3 of framework)
 # Keyword arguments:
+#   exp_results_import -- dictionary formatted experimental results
+#   sim_results_import -- dictionary formatted simulated results
 #   mofs_list -- list of MOF structures simulated
-#   experimental_mass_import -- dictionary formatted experimental results for each mof
 #   mof_densities -- dictionary of densities
 #   gases -- list of gases in simulated mixtures
-def import_experimental_results(mofs_list, exp_results_import, mof_densities, gases):
+def import_experimental_results(exp_results_import, mofs_list, mof_densities, gases):
```

Comment:
you reordered the variable definitions, but not the calling functions; those should be in the same commit.